src/pkgmt/versioneer.py

```diff
@@ -98,26 +105,16 @@ def version(
 
     Notes
     -----
-    Create a new version (projects with setup.py) :
-    1. version_package will be None
-    2. Set new stable version in package_name/__init__.py
-    3. Update header in CHANGELOG file, and ask to review CHANGELOG
-    4. Create commit for new version, create git tag, and push
-    5. Set new development version in package_name/__init__.py, and CHANGELOG
-    6. Commit new development version and push
-
-    Create a new version (projects without setup.py) :
-    1. These projects should contain two essential files:
-       config.yaml in root directory which should contain the repo
-       name
-       _version.py file containing __version__ in the required directory
-    2. version_package will be the directory containing _version.py file
-    3. Set new stable version in package_name/_version.py
-    4. Update header in CHANGELOG file, and ask to review CHANGELOG
-    5. Create commit for new version, create git tag, and push
-    6. Set new development version in package_name/_version.py, and CHANGELOG
-    7. Commit new development version and push
+    .. deprecated:: 0.6.2
+        `version_package argument is deprecated, will be removed in version 0.6.3
     """
+
+    if version_package is not None:
+        raise AttributeError(
+            "version_package attribute is deprecated. "
+            "Please add `version_file` in pyproject.toml file."
+        )
+
```

Comment:
The functionality is same as adding the version_file path in `pyproject.toml`.  If the invoker is not being used elsewhere I think it would be fine to remove it. It was added for the old cloud backend project.
@tonykploomber

Reply:
we can remove this